assets/stylesheets/nhsuk-print.scss

```diff
@@ -50,12 +60,82 @@ a {
   }
 }
 
+img {
+  max-width: 15cm;
+}
+
+.card__caption {
+  font-size: .9em;
+  font-style: italic;
+  margin-top: 0;
+}
+
 select {
   background: $white;
 }
 
+hr {
+  border: 1mm solid $grey-3;
+  margin: .8cm 0;
+  width: 15%;
+}
+
+.global-header__link::after {
+  content: none !important;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 1cm 0 .5cm;
+  page-break-after: avoid;
+}
+
+p,
+figure {
+  margin: .5cm 0;
+}
+
+details {
+  margin: 1cm 0;
+
+  summary {
+    @include bold-font(24);
+    margin-bottom: .5cm;
+
+    &::-webkit-details-marker {
+      display: none;
+    }
+  }
+}
+
+.list--cross li {
+  &::before,
+  &::after {
+    border: 2px solid $red;
+    height: 0;
+  }
+}
+
+.tabs-panel,
+.callout {
+  page-break-inside: avoid;
+}
+
+.callout--compact {
+  display: inline-block;
+}
+
 // hide the unnecessary page elements
+// sass-lint:disable class-name-format
 body footer,
-.visuallyhidden {
+.visuallyhidden,
+.skiplinks,
+.notification-banner,
+.local-header .link-list,
+.js-tabs-nav {
```

Comment:
The _Tell us what you think_ link is still displayed at the bottom of the page.

Wondering if it might be better to have a utility class that hides in print view so it's also clear when looking at the markup as to which elements we're doing something with them specifically on print?